### DISCUSSION TITLE
Error in AUCell_plotTSNE when running runSCENIC4

### DISCUSSION THREAD

**Discussion by ianfolkert** [aertslab/SCENIC#265] [R]
Hi, 

I just wanted to first say thank you for building this package, it has been incredibly useful in helping me to analyze some mouse tumor data where simply looking at cluster markers wouldn't have shown the full picture. I'm guessing there is something simple I am doing wrong, but when I run runSCENIC4 I get the following warning message: 

```
Binary regulon activity: 314 TF regulons x 8509 cells.
(314 regulons including 'extended' versions)
314 regulons are active in more than 1% (85.09) cells.
RStudioGD 
        2 
Warning message:
In AUCell_plotTSNE(tSNE = tSNE, exprMat = exprMat, cellsAUC = regulonAUC,  :
  Expression plot was requested, but no expression matrix provided.
```

I tried renaming my log-transformed expression matrix ot simply exprMat, but still can't seem to get rid of the warning above. Thanks for you help!

**Comment by KatjaRM** [comment]
I had the same issue but had done the mistake of deleting a file I should not have deleted. Did you forget running a previous step or clean up your folders?

**Comment by AAA-3** [comment]
> I had the same issue but had done the mistake of deleting a file I should not have deleted. Did you forget running a previous step or clean up your folders?

What was the file? I have had the same issue. I initially removed the original `exprMatrix` after the creation of `exprMatrix_filtered` (like it said in the tutorial) but even when I re-ran the code and kept both files, it wouldn't work.

**Comment by AAA-3** [comment]
FOLLOW UP:

I changed the code slightly after finding the [source code ](https://rdrr.io/github/aertslab/SCENIC/src/R/runSCENIC_4.R):
```
runSCENIC_4_aucell_binarize <- function(scenicOptions, 
                                        skipBoxplot=FALSE, skipHeatmaps=FALSE, skipTsne=FALSE, exprMat=NULL)
```
I edited my code to: 
`scenicOptions <- runSCENIC_4_aucell_binarize(scenicOptions, exprMat=exprMat_filtered)`
and it worked!
